### ISSUE TITLE
A task was canceled.

### ISSUE THREAD

**Issue by DoogeJ** [RatScanner/RatScanner#113]
![image](https://user-images.githubusercontent.com/4255780/127326091-590108f4-532e-40dd-b1ae-ba224851eb6a.png)

**Error**
A task was canceled.
```
System.Threading.Tasks.TaskCanceledException: A task was canceled.
   at System.Windows.Threading.DispatcherOperation.Wait(TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.InvokeImpl(DispatcherOperation operation, CancellationToken cancellationToken, TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.Invoke(Action callback, DispatcherPriority priority, CancellationToken cancellationToken, TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.Invoke(Action callback)
   at RatScanner.RatScannerMain.NameScan(Vector2 position)
   at RatScanner.HotkeyManager.Wrap[T](Func`1 func)
```
<details>
<summary>Log</summary>

```
[12:56:26.2162658] > [Info]  ----- RatScanner 2.7.0 -----
[12:56:26.2266651] > [Info]  Starting RatScanner...
[12:56:26.2267652] > [Info]  Loading config...
[12:56:26.2734182] > [Info]  Loading item data...
[12:56:26.2857307] > [Info]  Loading resource "RSItemDataVersion"...
[12:56:28.2952393] > [Info]  Loading TarkovTracker...
[12:56:29.1354734] > [Info]  Loading progress data...
[12:56:29.5004212] > [Info]  Loading price data...
[12:56:30.1742542] > [Info]  Setting default item...
[12:56:30.1749718] > [Info]  Initializing name scan...
[12:56:30.1754827] > [Info]  Loading OCR data...
[12:56:31.3426797] > [Info]  Loading TarkovTracker...
[12:56:32.3206293] > [Info]  Loading progress data...
[12:56:32.4578415] > [Info]  Loading price data...
[12:56:33.1657739] > [Info]  Setting default item...
[12:56:33.1665952] > [Info]  Initializing name scan...
[12:56:33.1672237] > [Info]  Loading OCR data...
[12:56:33.3604887] > [Info]  Initializing icon scan...
[12:56:33.3675022] > [Info]  Loading static icons...
[12:56:33.3701849] > [Info]  Initializing icon scan...
[12:56:33.3717072] > [Info]  Loading static icons...
[12:56:34.6004468] > [Info]  Loaded 1197/1197 icons successfully.
[12:56:34.6017012] > [Info]  Loading static correlation data...
[12:56:34.6017004] > [Info]  Loading static correlation data...
[12:56:34.6210309] > [Info]  Loading dynamic correlation data...
[12:56:34.6345059] > [Info]  Loading dynamic correlation data...
[12:56:34.6347728] > [Info]  Loading dynamic icons...
[12:56:34.6466889] > [Info]  Loading dynamic icons...
[12:56:34.6620724] > [Error] Unhandled exception in RatScanner 2.7.0
 -------------------- 
 System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (The process cannot access the file 'C:\Users\noxious\Downloads\RatScanner\Log.txt' because it is being used by another process.)
 ---> System.IO.IOException: The process cannot access the file 'C:\Users\noxious\Downloads\RatScanner\Log.txt' because it is being used by another process.
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.StreamWriter.ValidateArgsAndOpenPath(String path, Boolean append, Encoding encoding, Int32 bufferSize)
   at System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding)
   at System.IO.File.AppendAllText(String path, String contents, Encoding encoding)
   at RatScanner.Logger.ProcessBacklog()
   at RatScanner.Logger.<>c.<AppendToLog>b__12_0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__274_0(Object obj)
   at System.Threading.ExecutionContext
```
</details>

**Comment by DoogeJ** [comment]
(I was trying to close it)

**Comment by Blightbuster** [comment]
All those windows were opened by a single ratscanner instance?

**Comment by Blightbuster** [comment]
Fixed with v2.8.5
